### ISSUE TITLE
GalacticCraft map empty after launching rocket under GraalVM

### ISSUE THREAD

**Issue by lynxx131** [GTNewHorizons/GT-New-Horizons-Modpack#17681]
### Your GTNH Discord Username

Lynxx131

### Your Pack Version

2.7.0-Beta2

### Your Server

private server

### Java Version

Java 21

### Type of Server

Vanilla Forge

### Your Expectation

When launching a rocket the galaxy map should display planets and moons.

### The Reality

After the map opens I get this:

![2024-10-19 23_10_25-GT_ New Horizons 2 7 0](https://github.com/user-attachments/assets/cf61dc55-8ab4-49ee-a8c5-71cdaeedfdd5)

and this when selecting a planet from the menu:

![2024-10-19 23_19_09-GT_ New Horizons 2 7 0](https://github.com/user-attachments/assets/7727ae64-776f-45cf-865c-935d73759060)

Tested this on my server but also in a local world on a completely vanilla Java 21 install from the GTNH download site.

### Your Proposal

Not sure what's happening here

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by dagger8243** [comment]
in that 2nd screenshot is camera still moving along ?

**Comment by lynxx131** [comment]
Yeah the camera is still moving along it's orbit just no planet or moon.

**Comment by lynxx131** [comment]
Ah this appears to be a GraalVM related issue, switching to Adoptium JDK 21 resolved the issue so all good.

I was running `graalvm-ee-java17-22.3.5`, I don't know the dev team's stance on GraalVM so feel free to close this issue if it's not "officially" supported.

**Comment by lynxx131** [comment]
For reference I was running with the following GraalVM args `-Dgraal.LoopRotation=true -Dgraal.PartialUnroll=true -Dgraal.VectorizeSIMD=true -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1`

**Comment by lynxx131** [comment]
So if I remove `-Dgraal.VectorizeSIMD=true` then the map view works again.

Not sure if this is something that can be fixed or whether it's just a weird behaviour with Graal and GTNH.

If it's just an incompatibility it could be worth just noting it down somewhere.
